DogeNet.BLL/Features/Friends/DeleteFriend/DeleteFriandHandler.cs

```diff
@@ -0,0 +1,40 @@
+﻿// <copyright file="DeleteFriandHandler.cs" company="Leobit">
+// Copyright (c) Leobit. All rights reserved.
+// </copyright>
+
+namespace DogeNet.BLL.Features.Friends.DeleteFriend
+{
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using DogeNet.DAL;
+    using MediatR;
+
+    public class DeleteFriandHandler : IRequestHandler<DeleteFriendCommand>
+    {
+
+        private readonly DBContext context;
+
+        public DeleteFriandHandler(DBContext context)
+        {
+            this.context = context;
+        }
+
+        public Task<Unit> Handle(DeleteFriendCommand request, CancellationToken cancellationToken)
+        {
+            var list = this.context.Friends.Where(source =>
+            (source.UserId == request.model.UserId && source.FriendId == request.model.FriendId)
```

Comment:
Strange document formatting. Do you installed some formatter or code style analyzer that could handle that?

Reply:
done